File: Medii de Proiectare si Programare/HW/MPP_CSharp/MPP_CSharp/Domain/ChildTrackDTO.cs
namespace MPP_CSharp.Domain;

public class ChildTrackDTO
{
    public Child Child { get; set; }
    public int TrackCount { get; set; }

    public ChildTrackDTO(Child child, int trackCount)
    {
        this.Child = child;
        this.TrackCount = trackCount;
    }
    
}

File: Medii de Proiectare si Programare/HW/mpp_hw_copy/mpp_hw/TransportModel/src/main/java/transport/model/ChildTrackDTO.java
package transport.model;

public class ChildTrackDTO {
    private Child child;
    private int trackNumber;

    public ChildTrackDTO(Child child, int trackNumber) {
        this.child = child;
        this.trackNumber = trackNumber;
    }

    public Child getChild() {
        return child;
    }

    public void setChild(Child child) {
        this.child = child;
    }

    public int getTrackNumber() {
        return trackNumber;
    }

    public void setTrackNumber(int trackNumber) {
        this.trackNumber = trackNumber;
    }

}
